frontend/src/pages/Roster.js

```diff
@@ -0,0 +1,173 @@
+import React from "react";
+import MaterialTable from "@material-table/core";
+import { Helmet } from "react-helmet";
+import { Grid, Button } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { useDispatch } from "react-redux";
+import { TableIcons } from "../components/Icons";
+import LoadingContainer from "../components/LoadingContainer";
+import PageContainer from "../components/PageContainer";
+import { openAlert } from "../actions";
+import { getUsers } from "../services/users";
+import { withAuthorization } from "../components/HOC";
+
+const useStyles = makeStyles((theme) => ({
+  grid: {
+    textAlign: "center",
+  },
+  dates: {
+    marginBottom: theme.spacing(2),
+  },
+}));
+
+const styleButton = {
+  color: "inherit",
+  fontWeight: "900",
+  textDecoration: "underline",
+  textTransform: "none",
+  background: "transparent",
+};
+const deafultButton = {
+  color: "gray",
+  fontWeight: "900",
+  textTransform: "none",
+};
+
+const Roster = () => {
+  const classes = useStyles();
+  const [state, setState] = React.useState({
+    // Initial backend data
+    loading: true,
+    developers: [],
+    alumni: [],
+  });
+  const [selectedRow, setSelectedRow] = React.useState(null);
+  const [alumni, setAlumni] = React.useState(false);
+  const dispatch = useDispatch();
+
+  React.useEffect(() => {
+    const loadData = async () => {
+      const { ok, data } = await getUsers();
+      if (ok) {
+        const developers = data.users
+          .filter((user) => user.role.name !== "Alum")
+          .map((user) => ({
+            ...user,
+            role: user.role.name,
+          }));
+        const alums = data.users
+          .filter((user) => user.role.name === "Alum")
+          .map((user) => ({
+            ...user,
+            role: "Alum",
+          }));
+        setState((prev_state) => ({
+          ...prev_state,
+          loading: false,
+          developers,
+          alumni: alums,
+        }));
+      } else {
+        dispatch(openAlert(`Error: ${data.message}`));
+        setState((prev_state) => ({
+          ...prev_state,
+          loading: false,
+        }));
+      }
+    };
+    if (state.loading) {
+      loadData();
+    }
+  }, [state.loading, dispatch]);
+
+  return (
+    <>
+      <Helmet>
+        <title>TSE Roster</title>
+      </Helmet>
+      <PageContainer>
+        <LoadingContainer loading={state.loading}>
+          <Button
+            size="large"
+            style={alumni ? deafultButton : styleButton}
+            onClick={() => {
+              setAlumni(false);
+            }}
+          >
+            Active Members
```

Comment:
Can we change this text (and the "Alumni" text) to match the font style on the Figma?